src/Schema/SchemaFinder.php

```diff
@@ -106,9 +106,9 @@ public function invalidateCache( DIProperty $property, ChangeRecord $changeRecor
 	 *
 	 * @param DataItem $dataItem
 	 *
-	 * @return SchemaList|[]
+	 * @return ?SchemaList
```

Comment:
Is that actually a valid doc? Never seen that. Always like `SchemaList|null`. Or omitted in modern PHP, since you can express it in code.

Reply:
A quick search of the docs does not show the nullable notation, but I'm sure I've used.

Or maybe I only thought I had because I have been leaving it off where I could.